Add more services to the rabbitmq-reset playbook

priteau opened the pull request:
This change adds services that we often deploy: Barbican, Blazar, CloudKitty, Designate, Manila and Octavia.

MoteHue commented on etc/kayobe/ansible/rabbitmq-reset.yml:
```diff
@@ -69,4 +69,4 @@
     # The following services can have problems if the cluster gets broken.
```
Nit: Given this is now a list of all the services we deploy which use RabbitMQ, the comment here and at the start of the file could be update to reflect this. (Right now it suggests only some of them imo)

priteau replied:
I have updated the comments.

priteau committed:
Add more services to the rabbitmq-reset playbook

This change adds services that we often deploy: Barbican, Blazar,
CloudKitty, Designate, Manila and Octavia.
